user-guide/Advanced_Functionality/Swarming/SwarmingFAQ.md

```diff
@@ -0,0 +1,71 @@
+---
+uid: SwarmingFAQ
+---
+
+# Frequently asked questions
+
+## Can I swarm X/Y/Z?
+
+Swarming will gradually become available for more and more functionality in DataMiner.
+
+As of DataMiner 10.5.1, only basic elements are supported.
+
+For a list of upcoming features, see [Upcoming features](xref:Swarming#upcoming-features).
+
+## Does the ID of an element change when it is swarmed?
+
+No, the identifier (DataMiner ID/element ID) of an element that is swarmed stays the same.
+
+Each element also stores where it is hosted, and that "Host ID" field is updated when the element is swarmed.
+
+## What does the **DataMinerID** mean of an element that swarmed?
```

Comment:
I think this explanation might be taking us a bit too far, and could potentially cause more confusion than anything else, because the ElementID alone hasn't been used for a very, very long time, and the DataMiner ID hasn't been fixed to the hosting Agent since DataMiner 8.5, when DELT was introduced. I think it might be better to just combine the info from this question (in a slightly shorter form) with the one above, and explain that the DataMiner ID refers to the Agent where the element was originally created instead of the hosting Agent.

Reply:
You are right but I added this because we have gotten his exact question multiple times. From internal and external people.

Its just an overal confusing topic, but in my opinion the best way to understand the current situation is to understand how we got here through the historical evolution.

However, I may be biased as only the very technical questions get through to us, so a more digestable format might be preferred for docs.